great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -63,21 +62,7 @@ def __init__(
         # the risk of cyclic import issues. If these anonymizers have been imported at any earlier point
         # in the program's lifetime, retrieval of the import will be O(1) and not impact performance.
```

Comment:
Looks like you can remove these 3 comments

Reply:
done! great catch 🙇🏽